docs/how-to/configure.md

```diff
@@ -85,11 +85,16 @@ As well as via the *PMM Settings* page, you can also disable telemetry with the
 > - If the Security Threat Tool is enabled in PMM Settings, Telemetry is automatically enabled.
 >
 > - Telemetry is sent immediately; the 24-hour grace period is not honored.
+> 
+
+If you'll disable the already enabled Telemetry feature PMM will stop sending data and all feature where you can get some data from Percona will be not available to be enabled 
```

Comment:
```suggestion
>
> - Disabling telemetry prevents you from using features that require data from us.
```

Reply:
What features specifically are not available to the user when they disable telemetry? I have reworded an already vague sentence but it should be clearer.